src/common/components/help/help.css

```diff
@@ -11,6 +15,16 @@
   margin-bottom: 0;
 }
 
+.helpFlexWrapper {
```

Comment:
Some flex-box magic to make the headings of help boxes have equal height regardless of text length.